src/csharp/Microsoft.Spark/Sql/StorageLevel.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Data.SqlTypes;
+using System.Linq;
+using System.Net;
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+using Microsoft.Spark.Network;
+using Microsoft.Spark.Sql.Streaming;
+using Microsoft.Spark.Sql.Types;
+
+namespace Microsoft.Spark.Sql
+{
+    /// <summary>
+    /// Flags for controlling the storage of an RDD. Each StorageLevel records whether to use
+    /// memory, whether to drop the RDD to disk if it falls out of memory, whether to keep the
+    /// data in memory in a JAVA-specific serialized format, and whether to replicate the RDD
+    /// partitions on multiple nodes.Also contains static constants for some commonly used storage
```

Comment:
this will be > 100 char, so format as needed.
```suggestion
    /// partitions on multiple nodes. Also contains static properties for some commonly used storage
```